include/oneapi/dpl/internal/random_impl/linear_congruential_engine.h

```diff
@@ -122,7 +128,7 @@ class linear_congruential_engine
         state_ = mod_scalar(state_);
     }
 
-    template <int _N = 0, scalar_type _INC = 0>
+    template <int _N = 0>
     typename ::std::enable_if<(_N != 0)>::type
```

Comment:
could be _N > 0 for consistency with others checks?

Reply:
Fixed